tests/integration-all/s3/service/utils.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function getMarkers(functionName) {
+  return {
+    start: `--- START ${functionName} ---`,
+    end: `--- END ${functionName} ---`,
+  };
+}
+
+function log(functionName, message) {
+  const markers = getMarkers(functionName);
+  process.stdout.write(markers.start);
+  process.stdout.write(message);
+  process.stdout.write(markers.end);
```

Comment:
If it's supposed to be run in AWS lambda, then using `process.stdout.write` over `console.log` has implication of log being written out of convention (no request id and no timestamp)

Is that intentional?

Reply:
AFAIK it won't impact the correctness here. I'll update it regardless since it seems to be more intuitive to use `console.log` in that case.